integration-tests/fake_spine/fake_spine/config.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+
+class ConfigurationException(Exception):
+    pass
+
+
+def environ_or_error(key: str, default: str = None) -> str:
+    value = os.environ.get(key, default)
+    if not value:
+        raise ConfigurationException(f'The environment variable {key} is required.')
+    return value
+
+
+def eniron_or_error_as_int(key: str, default: str = None) -> int:
+    return int(environ_or_error(key, default))
```

Comment:
this seems redundant

Reply:
About half of the variable need conversion to into

Reply:
I saw that but 
`int(environ_or_error(key, default))`
`eniron_or_error_as_int(str, default)`
looks the same for me